doc/source/cluster/running-applications/job-submission/quickstart.rst

```diff
@@ -76,6 +76,11 @@ To tell the Ray Jobs CLI how to find your Ray Cluster, we will pass the Ray Dash
 
 Alternatively, you can also pass the ``--address=http://127.0.0.1:8265`` flag explicitly to each Ray Jobs CLI command, or prepend each command with ``RAY_ADDRESS=http://127.0.0.1:8265``.
 
+Additionally, if you wish to pass headers per HTTP request to the Cluster, then utilize the `RAY_JOB_HEADERS` environment variable.` Must be in JSON form.
+
+.. code-block:: bash
+    $ export RAY_JOB_HEADERS='{"KEY": "VALUE"}'
```

Comment:
Might just need a newline after the code block? But best to check how it's done in other parts of the doc

Reply:
Yep. That's it! Updated!